parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -44,6 +47,7 @@ type ProofBranch = Vec<H256>;
 type ForkVersion = [u8; 4];
 
 #[derive(Clone, Default, Encode, Decode, PartialEq, RuntimeDebug, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
```suggestion
#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
```

Then we drop the serde imports on line 23.